src/docs/product/issues/issue-details/performance-issues/uncompressed-asset/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Uncompressed Asset"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/uncompressed-asset/
+description: "Learn more about Uncompressed Asset issues and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Uncompressed asset issues are when one of the files your users are downloading to load your browser page was not compressed when transferred. This likely indicates a misconfiguration of the server or CDN (content delivery network) serving the asset file.
+
+## Detection Criteria
+
+The detector for this performance issue looks for specific asset (resource) spans with the following criteria:
+
+- The span has to be one of either `resource.link` or `resource.script` spans, which in practice means it is a `.css` or `.js` file.
+- It has to be over `512kb` in size, to rule out smaller files less likely to benefit from being compressed.
+- The span duration has to be over `500ms`, to rule out faster transfers or assets served in internal networks.
+- There has to be another asset span in the transaction that has compressed files. This helps rule out cases whether your users browser is not sending the `Accept-Encoding` header whatsoever.
+
+If Sentry isn't detecting an uncompressed asset issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```

Comment:
```suggestion
- It has to be over `512kb` in size, (to filter out smaller files that are less likely to benefit from being compressed).
- The span duration has to be over `500ms`, (to filter out faster transfers or assets served in internal networks).
- There has to be another asset span in the transaction that has compressed files. This helps filter out cases where a users' browser isn't sending the `Accept-Encoding` header.

If Sentry isn't detecting an uncompressed asset issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```